translate/lang/data.py

```diff
@@ -24,6 +24,7 @@
 import gettext
 import locale
 import os
+import pycountry
```

Comment:
These and associated uses need to operate conditionally and just pass through if pycountry isn't available.